src/wallets/hooks.ts

```diff
@@ -115,21 +115,14 @@ export function useAllWallets() {
 }
 
 export async function setActiveWallet(address?: string) {
-  // verify address
-
-  const [wallets] = useStorage<StoredWallet[]>(
```

Comment:
Bug: Hook called from a regular function, violates the rules of hooks when called.